src/main/resources/com/google/api/codegen/java/discovery_fragment.snip

```diff
@@ -66,11 +66,18 @@
         JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
 
         {@ApiName} {@serviceName()} = new {@ApiName}.Builder(httpTransport, jsonFactory, credential)
-          .setApplicationName("Google Cloud Platform Sample")
-          .build();
+            .setApplicationName("Google Cloud Platform Sample")
+            .build();
 
         {@initialize(method)}
         {@methodCall(method)}
+        @if context.hasMediaDownload(method)
+
+          // {@TODO()} Download desired media content. For more information, see:
```

Comment:
Should we just say `Download media content. For more information, see:`
I'm not sure if there can be multiple media objects that are returned, or if there's only one
The documentation suggests there's one output stream